c++/src/RleEncoderV2.cc

```diff
@@ -124,7 +124,8 @@ void RleEncoderV2::write(int64_t val) {
         }
 
         if (fixedRunLength == MAX_LITERAL_SIZE) {
-            determineEncoding(option);
+            option.encoding = DELTA;
+            option.isFixedDelta = true;
```

Comment:
After this PR, the comments at line 134 will become obsolete.
> // case 2: variable delta run

Reply:
The return on line 131 has ended the handling of the fixed delta run case. I think the comment on line 134 is not obsolete.  The following is exactly how to handle variable delta run.